client/components/tinymce/plugins/wpcom-view/views/simple-payments/style.scss

```diff
@@ -21,18 +21,128 @@
gMzMuOSAyNS41MTcgTCAzNS45IDEyLjcxNyBDIDM2IDEyLjUxNyAzNS42IDEyLjExNyAzNS4yIDEyLjExNyBaIE0gMzAuMSAxOS4zMTcgQyAyOS43IDIxLjQxNyAyOC4xIDIyLjkxNyAyNS45IDIyLjkxNyBDIDI0LjggMjIuOTE3IDI0IDIyLjYxNyAyMy40IDIxLjkxNyBDIDIyLjggMjEuMjE3IDIyLjYgMjAuMzE3IDIyLjggMTkuMzE3IEMgMjMuMSAxNy4yMTcgMjQuOSAxNS43MTcgMjcgMTUuNzE3IEMgMjguMSAxNS43MTcgMjguOSAxNi4xMTcgMjkuNSAxNi43MTcgQyAzMCAxNy40MTcgMzAuMiAxOC4zMTcgMzAuMSAxOS4zMTcgWiI+PC9wYXRoPgogICAgPHBhdGggZmlsbD0iIzAwMzA4NyIgZD0iTSA1NS4xIDEyLjExNyBMIDUxLjQgMTIuMTE3IEMgNTEgMTIuMTE3IDUwLjcgMTIuMzE3IDUwLjUgMTIuNjE3IEwgNDUuMyAyMC4yMTcgTCA0My4xIDEyLjkxNyBDIDQzIDEyLjQxNyA0Mi41IDEyLjExNyA0Mi4xIDEyLjExNyBMIDM4LjQgMTIuMTE3IEMgMzggMTIuMTE3IDM3LjYgMTIuNTE3IDM3LjggMTMuMDE3IEwgNDEuOSAyNS4xMTcgTCAzOCAzMC41MTcgQyAzNy43IDMwLjkxNyAzOCAzMS41MTcgMzguNSAzMS41MTcgTCA0Mi4yIDMxLjUxNyBDIDQyLjYgMzEuNTE3IDQyLjkgMzEuMzE3IDQzLjEgMzEuMDE3IEwgNTUuNiAxMy4wMTcgQyA1NS45IDEyLjcxNyA1NS42IDEyLjExNyA1NS4xIDEyLjExNyBaIj48L3BhdGg+CiAgICA8cGF0aCBmaWxsPSIjMDA5Y2RlIiBkPSJNIDY3LjUgNC45MTcgTCA1OS43IDQuOTE3IEMgNTkuMiA0LjkxNyA1OC43IDUuMzE3IDU4LjYgNS44MTcgTCA1NS41IDI1LjcxNyBDIDU1LjQgMjYuMTE3IDU1LjcgMjYuNDE3IDU2LjEgMjYuNDE3IEwgNjAuMSAyNi40MTcgQyA2MC41IDI2LjQxNyA2MC44IDI2LjExNyA2MC44IDI1LjgxNyBMIDYxLjcgMjAuMTE3IEMgNjEuOCAxOS42MTcgNjIuMiAxOS4yMTcgNjIuOCAxOS4yMTcgTCA2NS4zIDE5LjIxNyBDIDcwLjQgMTkuMjE3IDczLjQgMTYuNzE3IDc0LjIgMTEuODE3IEMgNzQuNSA5LjcxNyA3NC4yIDguMDE3IDczLjIgNi44MTcgQyA3MiA1LjYxNyA3MC4xIDQuOTE3IDY3LjUgNC45MTcgWiBNIDY4LjQgMTIuMjE3IEMgNjggMTUuMDE3IDY1LjggMTUuMDE3IDYzLjggMTUuMDE3IEwgNjIuNiAxNS4wMTcgTCA2My40IDkuODE3IEMgNjMuNCA5LjUxNyA2My43IDkuMzE3IDY0IDkuMzE3IEwgNjQuNSA5LjMxNyBDIDY1LjkgOS4zMTcgNjcuMiA5LjMxNyA2Ny45IDEwLjExNyBDIDY4LjQgMTAuNTE3IDY4LjUgMTEuMjE3IDY4LjQgMTIuMjE3IFoiPjwvcGF0aD4KICAgIDxwYXRoIGZpbGw9IiMwMDljZGUiIGQ9Ik0gOTAuNyAxMi4xMTcgTCA4NyAxMi4xMTcgQyA4Ni43IDEyLjExNyA4Ni40IDEyLjMxNyA4Ni40IDEyLjYxNyBMIDg2LjIgMTMuNjE3IEwgODUuOSAxMy4yMTcgQyA4NS4xIDEyLjAxNyA4My4zIDExLjYxNyA4MS41IDExLjYxNyBDIDc3LjQgMTEuNjE3IDczLjkgMTQuNzE3IDczLjIgMTkuMTE3IEMgNzIuOCAyMS4zMTcgNzMuMyAyMy40MTcgNzQuNiAyNC44MTcgQyA3NS43IDI2LjExNyA3Ny40IDI2LjcxNyA3OS4zIDI2LjcxNyBDIDgyLjYgMjYuNzE3IDg0LjUgMjQuNjE3IDg0LjUgMjQuNjE3IEwgODQuMyAyNS42MTcgQyA4NC4yIDI2LjAxNyA4NC41IDI2LjQxNyA4NC45IDI2LjQxNyBMIDg4LjMgMjYuNDE3IEMgODguOCAyNi40MTcgODkuMyAyNi4wMTcgODkuNCAyNS41MTcgTCA5MS40IDEyLjcxNyBDIDkxLjQgMTIuNTE3IDkxLjEgMTIuMTE3IDkwLjcgMTIuMTE3IFogTSA4NS41IDE5LjMxNyBDIDg1LjEgMjEuNDE3IDgzLjUgMjIuOTE3IDgxLjMgMjIuOTE3IEMgODAuMiAyMi45MTcgNzkuNCAyMi42MTcgNzguOCAyMS45MTcgQyA3OC4yIDIxLjIxNyA3OCAyMC4zMTcgNzguMiAxOS4zMTcgQyA3OC41IDE3LjIxNyA4MC4zIDE1LjcxNyA4Mi40IDE1LjcxNyBDIDgzLjUgMTUuNzE3IDg0LjMgMTYuMTE3IDg0LjkgMTYuNzE3IEMgODUuNSAxNy40MTcgODUuNyAxOC4zMTcgODUuNSAxOS4zMTcgWiI+PC9wYXRoPgogICAgPHBhdGggZmlsbD0iIzAwOWNkZSIgZD0iTSA5NS4xIDUuNDE3IEwgOTEuOSAyNS43MTcgQyA5MS44IDI2LjExNyA5Mi4xIDI2LjQxNyA5Mi41IDI2LjQxNyBMIDk1LjcgMjYuNDE3IEMgOTYuMiAyNi40MTcgOTYuNyAyNi4wMTcgOTYuOCAyNS41MTcgTCAxMDAgNS42MTcgQyAxMDAuMSA1LjIxNyA5OS44IDQuOTE3IDk5LjQgNC45MTcgTCA5NS44IDQuOTE3IEMgOTUuNCA0LjkxNyA5NS4yIDUuMTE3IDk1LjEgNS40MTcgWiI+PC9wYXRoPgo8L3N2Zz4K );
```

Comment:
With my `background-image` suggestion, I meant keeping the PNG as a separate file, but referencing it from CSS instead of JS. If I wanted to see how this inline image actually looks like, it's a bit harder than just viewing a PNG file.